src/containers/org/CollaborationsParticipationContainer.js

```diff
@@ -0,0 +1,97 @@
+// @flow
+import React, { useState } from 'react';
+
+import debounce from 'lodash/debounce';
+import styled from 'styled-components';
+import { List } from 'immutable';
+import { Colors, SearchInput, Typography } from 'lattice-ui-kit';
+
+import { World2Icon } from '../../assets';
+
+const { NEUTRAL } = Colors;
+
+const BasicListCard = styled.div`
+  align-items: center;
+  background-color: ${NEUTRAL.N50};
+  border-radius: 5px;
+  display: flex;
+  padding: 8px 16px;
+`;
```

Comment:
No change required. In addition to your `BasicListCard`, there also exists:

1. https://github.com/openlattice/lattice-orgs/blob/develop/src/containers/permissions/DataSetPermissionsCard.js#L26
![Screen Shot 2021-07-19 at 4 20 47 PM](https://user-images.githubusercontent.com/440299/126239578-823efbc5-9ebb-4b59-a2c4-db7c358df11f.png)


2. https://github.com/openlattice/lattice-orgs/blob/develop/src/containers/permissions/ObjectPermissionsCard.js#L58
![Screen Shot 2021-07-19 at 4 21 01 PM](https://user-images.githubusercontent.com/440299/126239581-76a344ca-dbd4-454c-92f8-035f781fc205.png)


It would be nice to consolidate this design into a common component in `src/components`.